build/source/netcdf/def_output.f90

```diff
@@ -125,7 +125,7 @@ subroutine def_output(summaVersion,buildTime,gitBranch,gitHash,nGRU,nHRU,nSoil,i
   fname   = trim(infile)//'_'//trim(fstring)//'.nc'
   call ini_create(nGRU,nHRU,nSoil,trim(fname),ncid(iFreq),err,cmessage)
   if(err/=0)then; message=trim(message)//trim(cmessage); return; end if
-  print*,'Created output file: '//trim(fname)//'; ncid = ', ncid(iFreq)
+  print*,'Created output file: '//trim(fname)//'   !; ncid = ', ncid(iFreq)
```

Comment:
I can see that this is a bug (semicolon should be a comma? and no double dash?). But a separate PR?

Reply:
yep, I wish I could say I put that there just to check that people were paying attention but the ' after // is a mistake.  The idea was to remove the ncid info which is informative if you happen to be in a debug mode on ncids, but not something a typical user would need to see all the time.

Reply:
I can correct this with another PR - I notice that when you're not running in hourly timestep mode, there's no progress output (eg telling the timestep).  I was going to fix that and send an update.

Reply:
You also do not need the second `//`

Reply:
right!